guides/project-process/commercial-terms.md

```diff
@@ -0,0 +1,22 @@
+# Commercial Terms
+
+## Invoice Terms for Professional Services
+
+We request a down-payment for the first 2 weeks to start working.
+
+After the first 2 weeks, invoices will be sent weekly on a Monday for the
+previous week’s work.
+
+## DVELP Professional Services Fees
+
+Our professional services are provided on a time and materials basis.
+
+Fees are exclusive of the below expenses. These expenses are chargeable.
+
+* travel cost to and from client site
+* travel time to and from client site (chargeable at the hourly rate for
```

Comment:
I think we need to be a bit careful here. This would include the commute to and from work, which otherwise would not be chargeable. Perhaps we should add in either a couple of examples below, or specify a journey length.